version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "6.0.7-SNAPSHOT"
+ThisBuild / version := "7.0.0"
```

Comment:
`7.0.0-SNAPSHOT` please as `7.0.0` will be automatically set by sbt-release during release process

Reply:
good catch, addressed in [9fa088d](https://github.com/evolution-gaming/pubsub/pull/163/commits/9fa088d8f14da6aa79807901770b6dc6eaf24030)